tests/cases/fourslash/codeFixAddMissingAwait_propertyAccess2.ts

```diff
@@ -0,0 +1,15 @@
+/// <reference path="fourslash.ts" />
+////async function fn(a: Promise<{ x: string }>) {
+////  console.log(3)
+////  a.x;
+////}
+
+verify.codeFix({
+  description: ts.Diagnostics.Add_await.message,
+  index: 0,
+  newFileContent:
+`async function fn(a: Promise<{ x: string }>) {
+  console.log(3)
```

Comment:
Shouldn't this have `;` after console.log and on that line instead of next line ?

Reply:
This is actually the typical preferred style for non-semicoloners. See https://standardjs.com/rules.html#semicolons

Reply:
/cc @orta to confirm people actually do this?

Reply:
[Prettier does it this way](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOEA2cB0mIDmAFAGYCUA3ADpRECGA7nQJYwAE5RZNIANCBAAOMZumSg6AJ0kQGABSkI0yEHQBuEZgBM+IAEaS6YANZwYAZUFHmUAshiSArnH4ALGAFtMAdVes4aFZgcOZKrMxqrACeKmBoyvw2aHCSMHKGBB50yCR0mMn8AFZoAB4AQoYmZuZ0HnAAMjZwOXkFIMUl5jYE2ACKjhDwLfkuIFaSyZIqenR6cJi6gpI2MN7aMK7IABwADPxLEMnehoIqSwEpas38AI4D8OlCyih0aAC0UHBwWt+6knB3Zj-dJ0TLZJC5Eb8ZIeZjDNpobp9e7NCGtUYwWZrLQbZAAJn4DhYmG6AGEIB4siooNBriBHMkACqzZ6Q5IAX3ZQA), to which I give more credence than StandardJS anyway. I feel good about keeping it.

Reply:
Yep, confirming, this is what I'm used to